app/javascript/vue/tasks/gazetteers/new_gazetteer/components/DisplayList.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="vue-table-container">
+    <table class="vue-table">
+      <thead>
+        <tr>
+          <th class="word-keep-all">Shape</th>
+          <th class="word-keep-all">Coordinates</th>
+          <th class="word-keep-all">Type</th>
+          <th />
+        </tr>
+      </thead>
+      <TransitionGroup
+        name="list-complete"
+        tag="tbody"
+      >
+        <tr
+          v-for="item in list"
+          :key="item.uuid || 1 "
```

Comment:
Avoid to use literal values for `key`, the value here should be unique. You can generate a random id for these cases. Looking the condition, I assume some objects couldn't not have uuid, in those cases, if you're getting them from the backend, you could use || item.id, or if not the case, you can generate a uuid for everywhere you create the object that could be displayed here, even if they came from the backend, add uuid for each one before store the list